ssr/src/page/view_profile_redirect.rs

```diff
@@ -0,0 +1,32 @@
+use crate::component::canisters_prov::AuthCansProvider;
+use crate::utils::profile::ProfileDetails;
+use leptos::*;
+use leptos_router::Redirect;
+
+#[component]
+fn ProfileLoading() -> impl IntoView {
```

Comment:
lets use `FullScreenSpinner` instead of this?

Reply:
yes, that can make it better in when its loading

Reply:
lets remove this component since its no longer required?